client/template.go

```diff
@@ -58,6 +58,7 @@ type Template struct {
 	FileName             string           `json:"fileName,omitempty" tfschema:",omitempty"`
 	IsTerragruntRunAll   bool             `json:"isTerragruntRunAll"`
 	IsAzureDevOps        bool             `json:"isAzureDevOps" tfschema:"is_azure_devops"`
+	GitlabProjectId      int              `json:"gitlabProjectId,omitempty" tfschema:",omitempty"`
```

Comment:
Missing from the API. It will need to be fixed in the backend.
Adding it... once fixed it will work properly.